ui/clutter.js

```diff
@@ -222,6 +250,70 @@ function displayFollowing() {
     }
 }
 
+function updateFollowing(checkboxElem) {
+
+    var hash = checkboxElem.getAttribute("hash");
+    var following = App.follows[hash] === true;
+    var userElem = $(checkboxElem).parent().parent().parent();
+    var user = App.handles[hash];
+
+    var fn = function() {
+        getUserFollowingData(hash,function(audience,listening_to){
+            userElem.html(makeUserHTML(user, audience, listening_to));
+        });
+    };
+
+    if (following) {
+        unfollow(hash,fn);
+    } else {
+        follow(hash,fn);
+    }
+}
+
+function makeUserHTML(user, audience, listening_to) {
+    var following = App.follows[user.hash] === true ? "checked" : "";
+    return '<div class="user"> <span class="handle" title="'+user.hash+'">@'+user.handle+'</span> <span class="audience">'+audience+'</span> <span class="listening-to">'+listening_to+'</span> <span class="follow-button"><input hash="'+user.hash+'" onclick="updateFollowing(this);" type="checkbox" '+following+'></span> </div>';
+}
+
+function getUserFollowingData(user_hash,fn) {
```

Comment:
So the whole idea here is to show for each user how many people they follow, and how many people follow them?

Reply:
Yah, that was the idea.  Actually the whole list is retrived so we could even make it a link which when you click on it, goes and actually shows them, but I didn't do that yet...